### PR TITLE
[risk=no] Clean up some duplicate DB mappings which could cause bugs

### PR DESCRIPTION
**Pull request by jmthibault79**
TODO: either merge these files or ensure that all have test coverage in StorageEnumsTest:
* DbStorageEnums
* DemographicSurveyEnum
* RdrExportEnums

(list generated by searching for BiMap)

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/db/model/DbStorageEnums.java [hunk 1]
```diff
@@ -321,25 +316,6 @@ public static Domain domainIdToDomain(String domainId) {
     return DOMAIN_ID_MAP.inverse().get(domainId);
   }
 
-  // Education
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/db/model/DbStorageEnums.java:324]: All of these are duplicated in DemographicSurveyEnum, often with different values.  Yikes!  These were not used, so let's kill them before they escape.

*** note: we have not done a definitive check as to whether these alternative values were used anywhere

File: api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java [hunk 2]
```diff
@@ -62,7 +62,11 @@ private Short enumToStorage(Object enumValue, Class enumClass)
       throws InvocationTargetException, IllegalAccessException {
     final Method enumToStorageMethod = enumClassToStorageMethod.get(enumClass);
     final Object returnValue = enumToStorageMethod.invoke(INDICATES_STATIC_METHOD, enumValue);
-    assertThat(returnValue instanceof Short).isTrue();
+
+    String invokedMethodDesc = String.format("%s(%s)", enumToStorageMethod.getName(), enumValue);
+
+    assertThat(returnValue).named(invokedMethodDesc).isNotNull();
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/db/model/StorageEnumsTest.java:68]: more informative errors

**Review by calbach — APPROVED** [review]
yikes, good find

File: api/src/main/java/org/pmiops/workbench/db/model/DbStorageEnums.java (hunk 1 above)
Comment (jmthibault79) [reply api/src/main/java/org/pmiops/workbench/db/model/DbStorageEnums.java:324]: Update: these were apparently never used

**Pull request merged**